app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
-    pass
+    assert sum(split_integer(8, 1)) == 8
 
 
 def test_should_split_into_equal_parts_when_value_is_divisible_by_parts():
-    pass
+    assert split_integer(6, 2) == [3, 3]
 
 
 def test_should_return_part_equals_to_a_value_when_slitting_into_one_part():
-    pass
+    assert split_integer(17, 1) == [17]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal():
-    pass
+    assert split_integer(3, 1) == sorted(split_integer(3, 1))
```

Comment:
Use explicit expected values to make code easier to read 
```suggestion
    assert split_integer(3, 1) == [3]
```
Also, this test case doesn't actually check if the list is sorted (it consists of only one element)